.eslintrc

```diff
@@ -28,6 +34,22 @@
           "__MONGOD__"
         ]
       }
-    ]
-  }
+    ],
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": "warn",
+    "@typescript-eslint/no-empty-function": "warn",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/no-var-requires": "off"
+  },
+  "overrides": [
```

Comment:
This is a cool pattern that I discovered: We can introduce more strict linting rules for .ts files. So the code quality increases by migrating to Typescript not only through the types but also through the linting. But the old code just works as before.

Reply:
Nice